cluster/juju/layers/kubernetes-e2e/layer.yaml

```diff
@@ -4,6 +4,9 @@ includes:
   - layer:tls-client
   - interface:http
 options:
+  basic:
+    packages:
+      - golang-ginkgo-dev
```

Comment:
Yeah, archive access is available behind the egress fw. which is a +1 to this strategy. My concern lies with drift, but until its a problem its unfounded.